.github/workflows/tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+on: pull_request
+
+jobs:
+  test-snap:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install Snapcraft
+        run: sudo snap install snapcraft --classic
+
+      - name: Install konf snap
+        run: |
+          /snap/bin/snapcraft --destructive-mode
```

Comment:
I'd maybe split this into a "Build konf snap" step

Reply:
I think you are right! Thank you